src/javascript/binary/websocket_pages/trade/price.js

```diff
@@ -27,8 +28,99 @@ const formatMoney          = require('../../common_functions/currency').formatMo
  * `Price.display()` to display the price details returned from server
  */
 const Price = (() => {
+    
+
     let type_display_id_mapping = {};
     let form_id                 = 0;
+    let Highcharts;
+    const sparkline_options = {
+        chart: {
+            type  : 'line',
+            margin: [2, 0, 2, 0],
+            width : 100,
+            height: 20,
+            style : {
+                overflow: 'visible',
+            },
+        },
+        title: {
+            text: '',
+        },
+        credits: {
+            enabled: false,
+        },
+        exporting: { enabled: false },
```

Comment:
can we make this style consistent? either one variable objects should be written in one line or have line break